src/server/services/buckets/routes.js

```diff
@@ -0,0 +1,66 @@
+import { allBucketsController } from '~/src/server/services/buckets/controllers/all.js'
+import { environmentBucketsController } from '~/src/server/services/buckets/controllers/environment.js'
+import { scopes } from '~/src/server/common/constants/scopes.js'
+import { authScope } from '~/src/server/common/helpers/auth/auth-scope.js'
+import { provideTabs } from '~/src/server/services/helpers/provide-tabs.js'
+import { provideSubNavigation } from '~/src/server/services/buckets/helpers/provide-sub-navigation.js'
+import { provideFormContextValues } from '~/src/server/common/helpers/form/provide-form-context-values.js'
+import { provideService } from '~/src/server/services/helpers/provide-service.js'
+
+const serviceTeamAndAdminUserScope = authScope([scopes.tenant, scopes.admin])
+
+const serviceBuckets = {
+  plugin: {
+    name: 'serviceBuckets',
+    register: (server) => {
+      server.ext([
+        {
+          type: 'onPreAuth',
+          method: provideService,
+          options: {
+            sandbox: 'plugin'
+          }
+        },
+        {
+          type: 'onPostHandler',
+          method: provideFormContextValues(),
```

Comment:
Theres no forms in this section? So no need for this `onPostHandler` that provides forms values